environment.py

```diff
@@ -238,19 +238,37 @@ def compute_reward(
             REWARD_CONFIG["height_limits"]["max_z"],
         )
 
+        exp_coef, subtraction_factor, max_diff_norm = (
+            REWARD_CONFIG["original_pos_reward"]["exp_coefficient"],
+            REWARD_CONFIG["original_pos_reward"]["subtraction_factor"],
+            REWARD_CONFIG["original_pos_reward"]["max_diff_norm"],
+        )
+
+        # MAINTAINING ORIGINAL POSITION REWARD
+        qpos0_diff = self.initial_qpos - state.qpos
+        original_pos_reward = jnp.exp(-exp_coef * jnp.linalg.norm(qpos0_diff)) - subtraction_factor * jnp.clip(
+            jnp.linalg.norm(qpos0_diff), 0, max_diff_norm
+        )
+
+        # HEALTHY REWARD
         is_healthy = jnp.where(state.q[2] < min_z, 0.0, 1.0)
         is_healthy = jnp.where(state.q[2] > max_z, 0.0, is_healthy)
```

Comment:
This one should be more about orientation and in exponent like 
```suggestion
        orientation = jnp.exp(-jnp.norm(self.projected_gravity[:, :2], dim=1) * weight)
```

Reply:
oooo cool